ui-v2/tests/acceptance/dc/acls/policies/navigate.feature

```diff
@@ -0,0 +1,15 @@
+@setupApplicationTest
+Feature: dc / policies / navigation
+  Scenario: Clicking a policy in the listing and back again
+    Given 1 datacenter model with the value "dc-1"
+    And 3 policy models
+    When I visit the policies page for yaml
+    ---
+      dc: dc-1
+    ---
+    Then the url should be /dc-1/acls/policies
+    And the title should be "Policies - Consul"
+    Then I see 3 policy models
+    When I click policy on the policies
+    And I click "[data-test-back]"
+    Then the url should be /dc-1/acls/policies

```

Comment:
Didn't we add the 'click create and back' tests in the other PRs also? Actually I just checked and I don't think we did it here either? https://github.com/hashicorp/consul/pull/8359/files#r461699456 ? Maybe just leave it for now and we can circle back later. Up to you.

Reply:
Oh yea, I just did create for intentions. Not for any fo the ACLs tabs.

Reply:
I'll leave those out for now.